### DISCUSSION TITLE
关于自定义权限控制的问题。

### DISCUSSION THREAD

**Discussion by Antar1on** [nonebot/discussions#197] [问答] [answered]
您好，请问nonebot2中，这个Permission有没有办法实现一个Permission(custom_check)and(superuser|group_admin)的形式，好像这儿只能用or不让用and。

**Comment by yanyongyu** [comment, marked as answer]
目前只能使用 `|` 哦~

如果你真的需要 `&` 关系，你可以参考 https://github.com/nonebot/nonebot2/blob/be459e0bbb2747826183448e3139435d8a1001b4/nonebot/internal/permission.py#L108-L128 这样强行 AND

**Reply by Antar1on** [reply]
好的，非常感谢您的帮助。
